### PR TITLE
Netobserv 332

### PR DESCRIPTION
**Pull request by memodi**
https://issues.redhat.com/browse/NETOBSERV-332
* Adding manifests for ServiceMonitor and ConfigMap  to send NO metrics to cluster prometheus 
* updating scripts to apply ServiceMonitor and ConfigMap manifests 
* Adding NO metrics file for kube-burner workload * updating Jenkinsfile to have python 3.9.12
* thanos URL and user workload secrets instead of prometheus for user workload metrics.
*  Create operator group
* Ability to deploy NOO from OperatorHub or from main branch

Note that: For this to work successfully, it is dependent on to get the correct token that's done by changes added to [forked e2e-benchmarking repo](https://github.com/memodi/e2e-benchmarking/blob/netobserv-trials/workloads/kube-burner/common.sh#L15-L20)

Test run from jenkins: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/memodi-e2e-benchmarking-multibranch-pipeline/job/NETOBSERV-332/21/

### CODE REVIEW

File: scripts/flows_v1alpha1_flowcollector_versioned.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    sampling: 0
+    cacheActiveTimeout: 5s
+    cacheMaxFlows: 1000
+    interfaces: []
+    excludeInterfaces: ["lo"]
+    logLevel: info
+  flowlogsPipeline:
+    kind: DaemonSet
+#    kind: Deployment
+#    replicas: 1
```
Comment (nathan-weinberg) [inline scripts/flows_v1alpha1_flowcollector_versioned.yaml:24]: Comments needed here?

Comment (memodi) [reply scripts/flows_v1alpha1_flowcollector_versioned.yaml]: I just added file as is NOO repo, pushed commit to remove them. thanks!

**Pull request merged**